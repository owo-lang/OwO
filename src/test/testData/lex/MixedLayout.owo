module A where
 a = do { b
     ; c ; e }
 b = c
 module C where {
     a
 ; b = do
   a <- do { c; d }
   return rua
 ; c
 }

