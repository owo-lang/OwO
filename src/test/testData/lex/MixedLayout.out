ModuleToken (0 1 1) (6 1 7)
IdentifierToken "A" (7 1 8) (8 1 9)
WhereToken (9 1 10) (14 1 15)
BraceLToken (16 2 2) (16 2 2)
IdentifierToken "a" (16 2 2) (17 2 3)
EqualToken (18 2 4) (19 2 5)
DoToken (20 2 6) (22 2 8)
BraceLToken (23 2 9) (24 2 10)
IdentifierToken "b" (25 2 11) (26 2 12)
SemicolonToken (32 3 6) (33 3 7)
IdentifierToken "c" (34 3 8) (35 3 9)
SemicolonToken (36 3 10) (37 3 11)
IdentifierToken "e" (38 3 12) (39 3 13)
BraceRToken (40 3 14) (41 3 15)
IdentifierToken "b" (43 4 2) (44 4 3)
EqualToken (45 4 4) (46 4 5)
IdentifierToken "c" (47 4 6) (48 4 7)
ModuleToken (50 5 2) (56 5 8)
IdentifierToken "C" (57 5 9) (58 5 10)
WhereToken (59 5 11) (64 5 16)
BraceLToken (65 5 17) (66 5 18)
IdentifierToken "a" (72 6 6) (73 6 7)
SemicolonToken (75 7 2) (76 7 3)
IdentifierToken "b" (77 7 4) (78 7 5)
EqualToken (79 7 6) (80 7 7)
DoToken (81 7 8) (83 7 10)
BraceLToken (87 8 4) (87 8 4)
IdentifierToken "a" (87 8 4) (88 8 5)
LeftArrowToken (89 8 6) (91 8 8)
DoToken (92 8 9) (94 8 11)
BraceLToken (95 8 12) (96 8 13)
IdentifierToken "c" (97 8 14) (98 8 15)
SemicolonToken (98 8 15) (99 8 16)
IdentifierToken "d" (100 8 17) (101 8 18)
BraceRToken (102 8 19) (103 8 20)
IdentifierToken "return" (107 9 4) (113 9 10)
IdentifierToken "rua" (114 9 11) (117 9 14)
SemicolonToken (119 10 2) (120 10 3)
IdentifierToken "c" (121 10 4) (122 10 5)
BraceRToken (124 11 2) (125 11 3)
BraceRToken (127 13 1) (127 13 1)
BraceRToken (127 13 1) (127 13 1)
